rfi_file_monitor/utils/__init__.py

```diff
@@ -45,16 +45,18 @@ def class_in_object_iterable(iterable: Iterable, klass) -> bool:
             return True
     return False
 
-def get_patterns_from_string(input: str,defaults =[]) -> List[str]:
-    if not input or not input.strip() and not defaults:
-        return ['*']
-    elif input and defaults:
-         return list(map(lambda x: x.strip(), input.split(','))) + defaults
-    elif not input and defaults:
-        return defaults
+def get_patterns_from_string(input: str, defaults: List =None) -> List[str]:
+    if not defaults:
+        if not input or not input.strip():
```

Comment:
I would flip the if and else block, and use `if input and input.strip():`

Reply:
done in d3d544b